Install.md

```diff
@@ -13,15 +13,14 @@ Great care was taken to avoid using any features of Java that would impact compa
 
 ## How to install
 On Mac OS X, installing via Homebrew is highly recommended.
-On Linux, if you have an RPM-based package manager, installing via the RPM is recommended.
-You will also soon have the option of installing via Linuxbrew.
+On Linux, you have the option of installing via Linuxbrew or an RPM-based package manager. Either is recommended.

```

Comment:
P3: I think we should recommend Linuxbrew until we have our RPM in various public repositories.  What do you think?